src/google/protobuf/io/io_win32.h

```diff
@@ -75,6 +76,24 @@ PROTOBUF_EXPORT int stat(const char* path, struct _stat* buffer);
 PROTOBUF_EXPORT int write(int fd, const void* buffer, size_t size);
 PROTOBUF_EXPORT std::wstring testonly_utf8_to_winpath(const char* path);
 
+enum ExpandWildcardsResult {
```

Comment:
I don't think this will work as it is--to get the scoping I believe it has to be written `enum class ExpandWildcardsResult`.

Reply:
Done.